src/usePlacesAutocomplete.ts

```diff
@@ -105,7 +105,7 @@ const usePlacesAutocomplete = ({
       }
 
       // @ts-expect-error
-      asRef.current.getPlacePredictions(
+      asRef?.current.getPlacePredictions(
```

Comment:
@kevenjesus The `current` property always exists in a ref, so I think the correct point of the `?` should be at `asRef.current?.getPlacePredictions`.

Reply:
thanks, i'm sending a new commit